jaxb-ri/pom.xml

```diff
@@ -117,6 +117,7 @@
         <mrjar.sourceDirectory>${project.basedir}/src/main/java-mr</mrjar.sourceDirectory>
         <base.java.level>7</base.java.level>
         <upper.java.level>9</upper.java.level>
+        <root.dir>${session.executionRootDirectory}/..</root.dir>
```

Comment:
what is the difference between this and '${maven.multiModuleProjectDirectory}' ?

Reply:
session.executionRootDirectory is supported since Maven 3.0.3

Reply:
ok, thanks